students/daniel_carrasco/session03/mailroom.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+def main():
+
+    print('\noptions\n')
+    print('A = Send a Thank You')
+    print('\n')
+
+    response = input('what do you want to do?').upper()
+    if response == 'A':
+        thankyou()
+    pass
+
+def thankyou():
+    response2 = input('Do you want to see a list?').lower()
+    if response2 == 'yes':
+        row_format ="{:>15}" * 3
+        print(row_format.format('Name','Donation','Amount'))
+        for name,amount,time in zip(donors,cash,times):
+            print(f'{name.title():>15}{"$ "+str(amount):>15}{time:>15}')
+
+    response3 = input('Add a name?').lower()
+    if response3 == 'yes':
+        response4 = input('Add first and last').title()
+        donors.append(response4)
+        response5 = input('Donation amount?')
```

Comment:
`response(x)` is not a good variable name.  It tells me nothing about what it is.  It should describe the value held within, like `donation_amount` or `first_and_last_name`